stream-handler/handlers/record_user_handler.py

```diff
@@ -0,0 +1,41 @@
+import boto3
+from tweet_type import TweetType
+import logging
+
+
+class RecordUserHandler:
+    def __init__(self):
+        _dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
+        self.table = _dynamodb.Table('Users')
+        self.log = logging.getLogger()
+
+    def handle(self, tweet: dict):
+        user = tweet.get('user')
+
+        response = self.table.update_item(
+            Key={
+                'id': user.get('id_str')
+            },
+            UpdateExpression=
+            'set '
+            'screen_name = :s, '
+            'profile_image_url = :p, '
+            'created_at = :c',
+            ExpressionAttributeValues={
+                ':s': user.get('screen_name'),
+                ':p': user.get('profile_image_url'),
+                ':c': self.__parse_datetime(user.get('created_at'))
+            },
+            ReturnValues='ALL_NEW'
+        )
+
```

Comment:
Is the approach to insert/update the user every time, no matter if they already exist or not? Just making sure I understand.